### PR TITLE
merge classic Hodgkin lymphoma, nodular sclerosis type

### LINKED ISSUE #7278: MONDO:0020327 & MONDO:0004665 nodular sclerosis classical Hodgkin lymphoma[Merge]
**Mondo term (ID and Label)**
MONDO:0020327 classic Hodgkin lymphoma, nodular sclerosis type

**Reason for deprecation**
These two records appear to be the same concept, both have exact matches that map to C0152268 in UMLS 
(Orphanet [98843](http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=98843) and SNOMEDCT [118608000](http://identifiers.org/snomedct/118608000) and NCI C3518) 

**Term to be merged with**
MONDO:0004665 nodular sclerosis classical Hodgkin lymphoma
**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7278

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -533101,8 +533097,7 @@ is_a: MONDO:0100546 {source="https://orcid.org/0000-0002-4142-7153|OMIM:620775"}
 id: MONDO:0958333
 name: thrombocytopenia 13, syndromic
 xref: OMIM:620776 {source="MONDO:equivalentTo"}
-is_a: MONDO:0100241 {source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"} ! inherited thrombocytopenia
-is_a: MONDO:0100241 {source="OMIM:620776"} ! inherited thrombocytopenia
+is_a: MONDO:0100241 {source="OMIM:620776", source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"} ! inherited thrombocytopenia
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:533100]: should this source be written like this: `source="https://orcid.org/0000-0002-4142-7153|OMIM:620776"` with the "|"?

**Review by twhetzel — CHANGES_REQUESTED** [review]
merge looks good, question on source formatting

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:533100]: that was from the ROBOT template, this is wrong, I'll fix - good catch, thanks!

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:533100]: done

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**